.github/workflows/node-versions.yml

```diff
@@ -52,17 +52,26 @@ jobs:
         run: npm run test
         working-directory: packages/blockchain
 
-      - name: Test Client
-        run: npm run test
-        working-directory: packages/client
+      # Temporarily disabled due to browser build error along node-versions CI run
+      # "TextDecoder is not defined" triggered in older Noder versions (Node 12/14)
+      # https://github.com/ethereumjs/ethereumjs-monorepo/pull/1305
+      # 2021-06-18
+      #
+      # TODO: give this another test run in times and reactivate (note that this
+      # might also be an issue which gets fixed downstream at some point in time)
+      # - name: Test Client
+      #   run: npm run test
+      #   working-directory: packages/client
```

Comment:
instead of disabling these tests completely, i will try adding `if: ${{ matrix.node >= 15 }}`

Reply:
awesome, looks like the skip worked:

<img width="373" alt="Screen Shot 2021-06-21 at 10 28 17 AM" src="https://user-images.githubusercontent.com/22116/122803399-68715400-d27b-11eb-9c84-d571538db3a1.png">

---

<img width="833" alt="Screen Shot 2021-06-21 at 10 26 42 AM" src="https://user-images.githubusercontent.com/22116/122803257-36f88880-d27b-11eb-813e-72b888565f0e.png">

---

<img width="841" alt="Screen Shot 2021-06-21 at 10 27 28 AM" src="https://user-images.githubusercontent.com/22116/122803303-4a0b5880-d27b-11eb-83d6-edbbb210fc92.png">

Reply:
Ah, great, yeah, that's a lot better, thanks for doing this! 😄 👍